docs/user_guide/model_repository.md

```diff
@@ -177,6 +190,19 @@ If the environment variables are set they will take a higher priority
 and will be used by Triton instead of the credentials set using the
 aws config command.
 
+By default, Triton makes a local copy of a remote model repository
+in a temporary folder, which is deleted after Triton server is shut down.
+If you would like to control where remote model repository is copied to,
+you may set the `TRITON_AWS_MOUNT_DIRECTORY` environment variable to
```

Comment:
I think we usually refer to Amazon storage as "S3", so it might be more appropriate to name the environment variable as `TRITON_S3_MOUNT_DIRECTORY` rather than `TRITON_AWS_MOUNT_DIRECTORY`.

Reply:
I was mostly inspired by what we use now, and environmental variables were using `AWS`

Reply:
Double checking: "AWS" is used on environment variables and "S3" on everywhere else, but the environment variable names (i.e. "AWS_SECRET_ACCESS_KEY") are specified/used by the S3 client initially, so I think it is up to you if you want to use "S3" or "AWS" on the environment variable name.